service/controller/resource/monitoring/pvresizingresource/delete.go

```diff
@@ -0,0 +1,11 @@
+package pvresizingresource
+
+import (
+	"context"
+)
+
+func (r *Resource) EnsureDeleted(ctx context.Context, obj interface{}) error {
+	// TODO Check if we need delete
```

Comment:
We do not really need to delete anything as the PV get's deleted by the prometheus operator, you can add a comment saying it and remove the TODO